bundles/org.openhab.binding.network/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -62,6 +62,15 @@
 					reduces the load when arping and Wake-on-LAN are used. All interfaces are used when left empty.</description>
 				<advanced>true</advanced>
 			</parameter>
+
+			<parameter name="assumeiOsDevice" type="boolean" required="true">
```

Comment:
I think is more clear to describe the functional behavior:

```suggestion
			<parameter name="wakeUpIOSDevice" type="boolean" required="true">
```

When it becomes configurable the `PROPERTY_IOS_WAKEUP` should also be removed.

Reply:
Removed the propertythe property

Reply:
Did you see the first part of the comment? 😉

Reply:
I think `assumeiOsDevice` is a good fit, but i am open to other sugestions. 
The opted  `wakeUpIOSDevice` could be interpreted as a function only limited to iOS devices, but it actually performs an extra port knock, to _all_ devices.

Maybe `useIOSWakeup` ?

Reply:
I think normally a space or - is used (wake up/wake-up) so `useIOSWakeUp` is also fine with me.

Reply:
Done